src/linearmixedmodel.jl

```diff
@@ -312,7 +312,7 @@ diagonal blocks from the conditional variance-covariance matrix,
     s² Λ(Λ'Z'ZΛ + I)⁻¹Λ'
 """
 function condVar(m::LinearMixedModel{T}) where {T}
-    L = m.L
+    return [condVar(m, fnm) for fnm in fnames(m)]
```

Comment:
This potentially recomputes the blocks to the right of `fname`  in `sparseL`  several times.

Reply:
Recomputing blocks to the right, which are smaller blocks, may not be the worst outcome.  The current version, with the recompute, is in the same ballpark as computing just the first block on the `mrk17_exp1` example
```julia
julia> @benchmark condVar($m6, :item)
BenchmarkTools.Trial: 13 samples with 1 evaluation.
 Range (min … max):  371.041 ms … 403.260 ms  ┊ GC (min … max): 0.52% … 0.48%
 Time  (median):     399.209 ms               ┊ GC (median):    0.48%
 Time  (mean ± σ):   396.114 ms ±   8.839 ms  ┊ GC (mean ± σ):  0.46% ± 0.14%

  ▁                               █        ▁ ▁         ██  ▁ ▁█  
  █▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁█▁▁▁▁▁▁▁▁█▁█▁▁▁▁▁▁▁▁▁██▁▁█▁██ ▁
  371 ms           Histogram: frequency by time          403 ms <

 Memory estimate: 75.74 MiB, allocs estimate: 4015.

julia> @benchmark condVar($m6, :subj)
BenchmarkTools.Trial: 336 samples with 1 evaluation.
 Range (min … max):  12.751 ms … 18.624 ms  ┊ GC (min … max): 0.00% … 11.51%
 Time  (median):     14.469 ms              ┊ GC (median):    0.00%
 Time  (mean ± σ):   14.869 ms ±  1.039 ms  ┊ GC (mean ± σ):  2.87% ±  5.10%

                                              █          ▆     
  ▄▁▁▁▁▆▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁█▁▁▁▁▇▁▁▁▁▁█▁▁▁ ▆
  12.8 ms      Histogram: log(frequency) by time      14.3 ms <

 Memory estimate: 14.73 MiB, allocs estimate: 1172.

julia> @benchmark condVar($m6)
BenchmarkTools.Trial: 13 samples with 1 evaluation.
 Range (min … max):  386.563 ms … 423.627 ms  ┊ GC (min … max): 0.50% … 0.53%
 Time  (median):     417.341 ms               ┊ GC (median):    0.56%
 Time  (mean ± σ):   415.208 ms ±   9.577 ms  ┊ GC (mean ± σ):  0.55% ± 0.03%

  ▁                                      █ ▁  ▁      ▁▁  █  █    
  █▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁▁█▁█▁▁█▁▁▁▁▁▁██▁▁█▁▁█▁▁ ▁
  387 ms           Histogram: frequency by time          423 ms <

 Memory estimate: 90.46 MiB, allocs estimate: 5192.
```
I suggest that we just leave this as it is and remove the rest of the function, which is vestigial and will not be executed.

Reply:
Works for me.